src/com/seafile/seadroid2/ui/fragment/ActivitiesFragment.java

```diff
@@ -192,7 +192,7 @@ private void viewFile(String repoID, String path) {
         intent.putExtra("filePath", path);
         intent.putExtra("account", getBrowserActivity().getAccount());
         intent.putExtra("taskID", taskID);
-        startActivity(intent);
+        startActivityForResult(intent, BrowserActivity.DOWNLOAD_FILE_REQUEST);
```

Comment:
If use `startActivityForResult` in Fragment, it will not call the `onActivityResult` of the parent Activity, so the file will not be opened after downloaded. 

The right way is to use `getBrowserActivity().startActivityForResult(intent, BrowserActivity.DOWNLOAD_FILE_REQUEST);`